docs/additional-resources/openms-git-workflow.md

```diff
@@ -0,0 +1,197 @@
+OpenMS Git Workflow
+===================
+
+Before getting started, install latest version of git to avoid problems like GitHub https authentication errors
+(see [Troubleshooting cloning errors](https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors) and a solution using [ssh](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories)).
```

Comment:
Thanks for including this! Figuring out this part is such a headache, pointing contributors to the relevant documentation right away is super useful and saves the contributor a lot of time and mental energy.